xcell/mappers/mapper_base_Planck.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+import healpy as hp
+import pymaster as nmt
+from .mapper_base import MapperBase
+
+
+class MapperBasePlanck(MapperBase):
+    def __init__(self, config):
+        self._get_Planck_defaults(config)
+        return
+
+    def _get_Planck_defaults(self, config):
+        self._get_defaults(config)
+        self.file_map = config['file_map']
+        self.file_hm1 = config.get('file_hm1', None)
+        self.file_hm2 = config.get('file_hm2', None)
+        self.file_mask = config.get('file_mask', None)
+        self.file_gp_mask = config.get('file_gp_mask', None)
+        self.file_sp_mask = config.get('file_sp_mask', None)
+        self.signal_map = None
+        self.hm1_map = None
+        self.hm2_map = None
+        self.diff_map = None
+        self.nl_coupled = None
+        self.cl_coupled = None
+        self.cls = {'Auto': None,
+                    'Cross': None}
+        self.custom_auto = True
+        self.mask = None
+        self.gal_mask_mode = config.get('gal_mask_mode', '60%')
+        self.gal_mask_modes = {'20%': 0,
+                               '40%': 1,
+                               '60%': 2,
+                               '70%': 3,
+                               '80%': 4,
+                               '90%': 5,
+                               '97%': 6,
+                               '99%': 7}
```

Comment:
Percentages?? I'd use decimal numbers. I'd not use symbols if avoidable

Reply:
Done